src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -481,15 +487,25 @@ private PaymentInfo buildPaymentInfo(final Account account, final UUID kbPayment
         final String ccType = PluginProperties.getValue(PROPERTY_CC_TYPE, paymentMethodsRecordCcType, properties);
         final String recurringDetailId = PluginProperties.findPluginPropertyValue(PROPERTY_RECURRING_DETAIL_ID, mergedProperties);
 
+        final PaymentInfo paymentInfo;
         if (recurringDetailId != null) {
-            return buildRecurring(paymentMethodsRecord, paymentProvider, mergedProperties);
+            paymentInfo = buildRecurring(paymentMethodsRecord, paymentProvider, mergedProperties);
         } else if ("sepadirectdebit".equals(ccType)) {
-            return buildSepaDirectDebit(nonNullPaymentMethodsRecord, paymentProvider, mergedProperties);
+            paymentInfo = buildSepaDirectDebit(nonNullPaymentMethodsRecord, paymentProvider, mergedProperties);
         } else if ("elv".equals(ccType)) {
-            return buildElv(nonNullPaymentMethodsRecord, paymentProvider, mergedProperties);
+            paymentInfo = buildElv(nonNullPaymentMethodsRecord, paymentProvider, mergedProperties);
         } else {
-            return buildCreditCard(nonNullPaymentMethodsRecord, paymentProvider, mergedProperties);
+            paymentInfo = buildCreditCard(nonNullPaymentMethodsRecord, paymentProvider, mergedProperties);
         }
+
+        paymentInfo.setContinuousAuthenticationEnabled(isContinuousAuthenticationEnabled(mergedProperties));
+
+        return paymentInfo;
+    }
+
+    private boolean isContinuousAuthenticationEnabled(final Iterable<PluginProperty> mergedProperties) {
+        final String contAuth = PluginProperties.findPluginPropertyValue(PROPERTY_CONTINUOUS_AUTHENTICATION, mergedProperties);
+        return Boolean.parseBoolean(contAuth);
```

Comment:
Shouldn't we set it to `true` by default? In a typical e-commerce shopping cart or recurring (subscription) transaction, re-entering the CVV is not required.